aries_cloudagent/core/conductor.py

```diff
@@ -467,10 +472,24 @@ async def outbound_message_router(
                 outbound.reply_from_verkey = inbound.receipt.recipient_verkey
             # return message to an inbound session
             if self.inbound_transport_manager.return_to_session(outbound):
-                return OutboundSendStatus.SENT_TO_SESSION
+                await profile.notify(
+                    OutboundStatusEvent.topic_root,
```

Comment:
This will not have the desired effect. We will need to either make `profile.notify` accept an event or use `event_bus.notify`.

Reply:
This follows for each `profile.notify` below.